django/db/models/query.py

```diff
@@ -353,6 +353,11 @@ def create(self, **kwargs):
         obj.save(force_insert=True, using=self.db)
         return obj
 
+    def populate_pk_values(self, objs):
```

Comment:
This method should be made private as it does not belong on `Manager`. I think this is the source of the test failures.